lang/en.xml

```diff
@@ -475,15 +475,15 @@ Are you sure you want to continue?</user_delete_warn>
 	</user_management>
 
 	<vita3k_update name="Vita3K Update">
- 		<new_version_available>A new version of the Vita3K is available.</new_version_available>
+ 		<new_version_available>A new version of Vita3K is available.</new_version_available>
 		<back>Back</back>
 		<downloading>Downloading...
 After the download is complete, Vita3K will restart automatically and then install the new Vita3K.</downloading>
 		<not_complete_update>Could not complete the update.</not_complete_update>
 		<next>Next</next>
 		<update_vita3k>Do you want to update Vita3K?</update_vita3k>
-		<later_version_already_installed>The later version of the Vita3K software is already installed.</later_version_already_installed>
-		<latest_version_already_installed>The latest version of the Vita3K is already installed.</latest_version_already_installed>
+		<later_version_already_installed>The later version of Vita3K software is already installed.</later_version_already_installed>
```

Comment:
deleted software on lang state but not on xml languages :thinking:

Reply:
OUPSY __